scss/partials/_menu.scss

```diff
@@ -0,0 +1,459 @@
+          }
+        }
+
+        &::before {
+          content: "";
+        }
+      }
+    }
+
+    &[open] {
+      position: absolute;
+      top: 0.5rem;
+      width: 100%;
+      @include breakpoint(large) {
+        position: relative;
+      }
+
+      .dropdown-menu {
+        transform: translateX(0);
+      }
+
+      summary {
+        padding: 0 2rem;
+
+        @include breakpoint(medium) {
+          padding: 0;
+        }
+
+        @include breakpoint(large) {
+          padding: 0;
+        }
+
+        span {
+          background-color: $secondary-blue-1;
+          display: block;
+          padding: 2rem;
+          @include breakpoint(large) {
+            background-color: transparent;
+          }
+        }
+
+        .arrow {
+          &--after {
+            display: none;
+            @include breakpoint(large) {
+              display: inline-block;
+              border-left: 0.5rem solid transparent;
+              border-right: 0.5rem solid transparent;
+              border-bottom: 0.5rem solid $primary-white;
+              border-top: none;
+            }
+          }
+
+          &--before {
+            display: block;
+
+            @include breakpoint(large) {
+              display: none;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .navbar {
+    position: absolute;
+    top: 6.5rem;
+    z-index: 1;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+    /* basic menu styling*/
+    list-style: none;
+    display: flex;
+    flex-flow: column nowrap;
+    // justify-content: center;
+    padding: 5rem 2rem 0 0;
+    background: $primary-blue;
+    /* animate slide left to right */
+    transform: translateX(-100%);
+    transition: transform 0.2s ease;
+
+    @include breakpoint(medium) {
+      width: 50%;
+    }
+
+    @include breakpoint(large) {
+      height: 100%;
+      width: 100%;
+      display: grid;
+      position: relative;
+      top: 0;
+      transform: none;
+      overflow: visible;
+      padding: 2rem;
+    }
+
+    &:target {
+      transform: translateX(0);
+      /* 
+        * Transform all the slices of hamburger
+        * into a crossmark.
+        */
+      & ~ .menu-toggle {
+        border-bottom: 0;
+
+        &:after,
+        &:before {
+          height: 0.2rem;
+          width: 100%;
+          top: 50%;
+          left: 0;
+          background: $primary-white;
+          transition: transform 250ms ease, background 250ms ease;
+        }
+
+        &:before {
+          transform: rotate(45deg);
+        }
+
+        &:after {
+          transform: rotate(-45deg);
+        }
+
+        a {
+          position: absolute;
+
+          &#close-menu {
+            display: inline-block;
+          }
+
+          &#open-menu {
+            display: none;
+          }
+        }
+      }
+    }
+  }
+}
+
+@include breakpoint(large) {
+  .menu-toggle {
+    display: none;
+  }
+}
+
+.show-for-sr {
+  display: none;
+}
```

Comment:
This should probably go in the util partial instead. It also reads kind weird. If it's `display:none;`, then it's not going to show for screenreaders. Screenreaders will skip hidden content.